godtools/App/Features/Onboarding/OnboardingQuickStartView.xib

```diff
@@ -109,8 +80,8 @@
         </designable>
     </designables>
     <resources>
-        <systemColor name="systemBackgroundColor">
-            <color white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+        <systemColor name="systemPurpleColor">
+            <color red="0.68627450980392157" green="0.32156862745098042" blue="0.87058823529411766" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
```

Comment:
What is this color change for?

Reply:
Hey @reldredge71 that systemPurpleColor is set on the bottomGradientView in the .xib.  It's okay to have it set there because it is going to be set programmatically when applying the gradient.  Sometimes I do that in the .xibs just to highlight elements and if they are going to have to be programmatically set.